src/pages/Categories/CategoriesBeta.vue

```diff
@@ -156,15 +171,40 @@ export default {
 		return {
 			categoryPlaceholderImageCTF: '',
 			categories: [],
+			pageData: {},
 		};
 	},
 	apollo: {
-		query: allCategoriesQuery,
-		preFetch: true,
-		result(result) {
-			this.categories = result.data?.lend?.loanChannels?.values ?? [];
+		preFetch(config, client) {
+			return client.query({
+				query: allCategoriesQuery
```

Comment:
You can combine both of these two queries together by just combining the Graphql query object so that they happen asynchronously instead of waiting for the categories query, then executing the contentful query